payu/models/cable.py

```diff
@@ -0,0 +1,151 @@
+"""payu.models.cable
+   ================
+
+   Driver interface to CABLE
+
+   :copyright: Copyright 2021 Marshall Ward, see AUTHORS for details
+   :license: Apache License, Version 2.0, see LICENSE for details
+"""
+
+# Standard Library
+import glob
+import os
+import shutil
+
+# Extensions
+import f90nml
+import yaml
+
+# Local
+from payu.fsops import mkdir_p
+from payu.models.model import Model
+
+
+def _get_forcing_path(variable, year, input_dir, offset=None, repeat=None):
+    """Return the met forcing file path for a given variable and year."""
```

Comment:
Some description of the arguments would be helpful. I see below that `offset` and `repeat` are 2-member tuples, but it isn't really obvious what they mean.